src/components/entity-panel/sections/entity/wot/TheEntityListContent.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="custom-the-entity-list-content">
+    <v-divider class="px-2"></v-divider>
+    <entity-search v-model="search" :game="game" placeholder="entity.textField.placeholder" />
+    <v-chip-group
+      :value="selectedDefaultEntity || undefined"
+      class="custom-default-entities-container"
+      column
+      @change="defaultEntityChipGroupOnChangeHandler"
+    >
+      <entity-card
+        v-for="(entity, i) in getDefaultEntities"
+        :key="i"
+        :entity="entity"
+      />
+    </v-chip-group>
+    <v-lazy
+      v-if="entitySearch.length > 0"
+      max-height="200"
+    >
+      <v-chip-group
+        :value="selectedEntity || undefined"
+        class="custom-entities-container px-1"
+        column
+        @change="entityChipGroupOnChangeHandler"
+      >
+        <entity-card
+          v-for="(entity, i) in entitySearch"
+          :key="i"
+          :entity="entity"
+        />
+      </v-chip-group>
+    </v-lazy>
+  </div>
+</template>
+
```

Comment:
Why is this containing wows stuff, while its for wot? Placeholding data for wot here should be added.

Reply:
We haven't developed the WOT stuff yet. This is just placeholder code for now